make_prg/subcommands/from_msa.py

```diff
@@ -1,23 +1,46 @@
-import logging
+from typing import List
+import multiprocessing
 from pathlib import Path
-
-from make_prg.from_msa import prg_builder, NESTING_LVL, MIN_MATCH_LEN
-from make_prg import io_utils
-from make_prg.subcommands.output_type import OutputType
+from loguru import logger
+from make_prg import prg_builder
+from make_prg.from_msa import NESTING_LVL, MIN_MATCH_LEN
+from make_prg.utils import io_utils, gfa
 
 
 def register_parser(subparsers):
     subparser_msa = subparsers.add_parser(
         "from_msa",
-        usage="make_prg from_msa [options] <MSA input file>",
+        usage="make_prg from_msa",
         help="Make PRG from multiple sequence alignment",
     )
-
     subparser_msa.add_argument(
-        "MSA",
+        "-i",
+        "--input",
         action="store",
         type=str,
-        help="Input file: a multiple sequence alignment",
+        required=True,
+        help=(
+            "Input data. If it is a file, will be read as the supported alignment_format (see "
+            "parameter --alignment_format). A single PRG, GFA and update data structures will "
+            "be created for the given MSA. If it is a dir, will process all files in the dir. "
+            "Multiple PRGs, GFAs and update data structures will be created, one for each MSA"
+        ),
+    )
+    subparser_msa.add_argument(
+        "-o",
+        "--output_prefix",
+        action="store",
+        type=str,
+        required=True,
+        help="Prefix for the output files",
+    )
+    subparser_msa.add_argument(
+        "-t",
+        "--threads",
+        action="store",
+        type=int,
+        default=1,
+        help="Number of threads (default 1)",
```

Comment:
```suggestion
        help="Number of threads Default: %(default)d",
```

See https://docs.python.org/3.6/library/argparse.html#help

Reply:
Agreed